style.css

```diff
@@ -0,0 +1,1427 @@
+  margin: auto;
+  display: flex;
+}
+
+.section-heading > a {
+  text-decoration: none;
+}
+
+.section-heading-left {
+  font-weight: 900;
+  color: #f43;
+}
+
+.section-heading-right {
+  padding-top: 0.8em;
+  color: #f43;
+}
+
+.funding-rounds {
+  height: auto;
+  margin: auto;
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 8vh;
+  width: 87%;
+}
+
+.funding-rounds p {
+  font-size: 78%;
+  color: #555;
+}
+
+.info-cards {
+  box-sizing: border-box;
+  height: 300px;
+  position: relative;
+  margin: 0 5px 0 5px;
+  width: 25%;
+}
+
+.info-cards-image {
+  height: 70px;
+  position: absolute;
+  top: 10px;
+  left: 10px;
+  width: 70px;
+}
+
+.info-cards a {
+  text-decoration: none;
+}
+
+.top-title-1 {
+  height: 50px;
+  text-align: center;
+  margin-top: -18px;
+  background-color: #f43;
+}
+
+.top-title-1 h3 {
+  color: white;
+  padding: 3px;
+}
+
+.top-title-2 {
+  text-align: center;
+  margin-top: -18px;
+  color: black;
+  background-color: #ffece7;
+}
+
+.main-section {
+  padding: 5px;
+  margin-top: -15px;
+  color: black;
+  background-color: rgb(248, 248, 248);
+}
+
+.last-section {
+  padding: 5px;
+  color: black;
+  border-top: 1px solid #fcd8cf;
+  background-color: rgb(248, 248, 248);
+}
+
+.great-mid {
+  height: auto;
+  margin: auto;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  column-gap: 20px;
+  width: 86%;
+  grid-template-areas:
+    "hard apps gear"
+    "tech creative podium"
+    "insights launch distract";
+  margin-bottom: 8vh;
+}
+
+.hard-fork {
+  display: flex;
+  flex-direction: column;
+}
+
+.apps {
+  display: flex;
+  flex-direction: column;
+  grid-area: apps;
+}
+
+.gear {
+  display: flex;
+  flex-direction: column;
+  grid-area: gear;
+}
+
+.tech {
+  display: flex;
+  flex-direction: column;
+  grid-area: tech;
+}
+
+.creative {
+  display: flex;
+  flex-direction: column;
+  grid-area: creative;
+}
+
+.podium {
+  display: flex;
+  flex-direction: column;
+  grid-area: podium;
+}
+
+.insights {
+  display: flex;
+  flex-direction: column;
+  grid-area: insights;
+}
+
+.launch {
+  display: flex;
+  flex-direction: column;
+  grid-area: launch;
+}
+
+.distract {
+  display: flex;
+  flex-direction: column;
+  grid-area: distract;
+}
+
+.hard-fork-p1 {
+  background-image:
+    linear-gradient(0deg, rgba(0, 0, 0, 1) 0%, rgba(255, 255, 255, 0) 70%),
+    url("./images/Gamestop.jpg");
+  height: 250px;
+  color: #fff;
+  position: relative;
+  background-size: cover;
+}
+
+.apps-p1 {
+  background-image:
+    linear-gradient(0deg, rgba(0, 0, 0, 1) 0%, rgba(255, 255, 255, 0) 70%),
+    url("./images/Gamestop.jpg");
+  background-size: cover;
+  height: 250px;
+  color: #fff;
+  position: relative;
+}
+
+.gear-p1 {
+  background-image:
+    linear-gradient(0deg, rgba(0, 0, 0, 1) 0%, rgba(255, 255, 255, 0) 70%),
+    url("./images/Gamestop.jpg");
+  background-size: cover;
+  height: 250px;
+  color: #fff;
+  position: relative;
+}
```

Comment:
## Aim to Stay `DRY`
- [ ] The lines of code from lines `#372` to `#452` appear to be similar and make you seem like you are repeating yourself. Would you mind merging them so as to not appear redundant? A good suggestion would be to have a new `class` for the repeating rule.